src/BinSkim.Rules/ElfRules/BA3002.DoNotMarkStackAsExecutable.cs

```diff
@@ -69,7 +69,7 @@ public override void Analyze(BinaryAnalyzerContext context)
                         context.Logger.Log(this,
                             RuleUtilities.BuildResult(FailureLevel.Error, context, null,
                                 nameof(RuleResources.BA3002_Error_StackExec),
-                                context.TargetUri.GetFileName()));
+                                context.CurrentTarget.Uri.GetFileName()));
```

Comment:
>CurrentTarget [](http://example.com/codeflow?start=40&length=13)

Craig, you need to properly merge this branch to get rid of changes that didn't originate with your work.